diff --git a/dist/typings/index.d.ts b/dist/typings/index.d.ts
index 0ba916306ed8fe3a4213338968074519f071e348..b00cf37973d123ddf5d5e7b9072f3449a8db3d93 100644
--- a/dist/typings/index.d.ts
+++ b/dist/typings/index.d.ts
@@ -158,4 +158,5 @@ export declare function expect<K, V>(obj: Map<K, V>): MapAssertion<K, V>;
 export declare function expect<K extends object, V>(obj: WeakMap<K, V>): WeakMapAssertion<K, V>;
 export declare function expect<V>(obj: Set<V>): SetAssertion<V>;
 export declare function expect<V extends object>(obj: WeakSet<V>): WeakSetAssertion<V>;
+export declare function expect<T extends object>(obj: T): ObjectAssertion<T>;
 export declare function expect<T>(obj: T): Assertion<T>;
